src/assets/assets.controller.ts

```diff
@@ -176,4 +177,34 @@ export class AssetsController {
 
     res.status(HttpStatus.OK).json({ missing });
   }
+
+  @ApiOperation({ summary: 'Lists all verified asset metadata' })
+  @Get('verified_metadata')
+  @Header('Cache-Control', 's-maxage=60, stale-if-error=60')
+  async verifiedMetadata(
+    @Req() req: Request,
+    @Res() res: Response,
+  ): Promise<void> {
+    const lastUpdate = await this.assetsService.lastMetadataUpdate();
+    if (lastUpdate) {
+      handleIfModifiedSince(lastUpdate, req, res);
+    }
+
+    const assetsMetadata = (await this.assetsService.listMetadata()).map(
+      (m) => {
+        return {
+          identifier: m.identifier,
+          symbol: m.symbol,
+          decimals: m.decimals,
```

Comment:
Probably should not put fields on here if they are `null` because the spec for verified asset metadata JSON shows these fields as optional but not nullable